addons/web_editor/static/tests/wysiwyg_keyboard_tests.js

```diff
@@ -2079,6 +2079,30 @@ var keyboardTestsComplex = [{
             start: "h1:contents()[0]->15",
         },
     },
+    {
+        name: "in ul > li > ul > 1st li > empty p (other ul li ul before): BACKSPACE at start",
+        content: '<ul><li><p>1</p></li><li class="o_indent"><ul><li><p>2</p></li></ul><ul class="o_checklist"><li id="checklist-id-1"><p><br></p></li></ul></li><li><p>3</p></li></ul>',
+        steps: [{
+            start: "p:eq(2)->0",
+            key: 'BACKSPACE',
+        }],
+        test: {
+            content: '<ul><li><p>1</p></li><li class="o_indent"><ul><li><p>2</p></li></ul></li><li><p>3</p></li></ul>',
+            start: "p:eq(1):contents()[0]->1",
+        },
+    },
+    {
+        name: "in ul.list-group (div > p > textbefore): BACKSPACE at start",
```

Comment:
Broken test in the current commit.

Reply:
Also `ul.list-group (div > p > textbefore): BACKSPACE at start (carret position)` test is also broken.